lib/dialects/abstract/registry.js

```diff
@@ -0,0 +1,36 @@
+
+const _registries = {};
```

Comment:
```suggestion
const _registries = Object.create(null);
```

so getDialect('toString') returns null and avoids prototype injection

Reply:
Can we call it registry? It's a single registry after all :)